src/components/treePage/siteImageCarousel.tsx

```diff
@@ -87,11 +112,31 @@ export const SiteImageCarousel: React.FC = () => {
                   'Anonymous',
               })}
             </div>
-            <div>
-              {latestEntrySiteImages[currSlideIndex].uploadedAt ||
-                t('site_image.no_upload_date')}
-            </div>
+            <Space size={15}>
+              <div>
+                {latestEntrySiteImages[currSlideIndex].uploadedAt ||
+                  t('site_image.no_upload_date')}
+              </div>
+              <div>
+                <DeleteSiteImageButton
+                  type="primary"
+                  onClick={() => setShowDeleteForm(!showDeleteForm)}
+                >
+                  Delete
+                </DeleteSiteImageButton>
+              </div>
```

Comment:
⛏️ Can we make it so this button only shows up for admins and the user who uploaded the image? The delete site image route is only callable by admins and the image's uploader - we probably shouldn't give other users this option if it won't work for them. 
The image uploader's ID is available from the image, and we can get it the same way we get the image's uploader username or upload date (this field might need to be added to the `SiteEntryImage` type)